compiler/surface/lexer_common.mli

```diff
@@ -45,6 +45,16 @@ val calc_precedence : string -> int
 val get_law_heading : Sedlexing.lexbuf -> Tokens.token
 (** Gets the [LAW_HEADING] token from the current [lexbuf] *)
 
+(** Simplified tokens for dependency extraction *)
+type line_token =
```

Comment:
I would have called it `dependency_line_token`

Reply:
I didn't change the name in the end because by itself it's not so strictly related with dependencies; it just recognises some kinds of lines like inclusion and module directives, but also the ones delimiting code blocks (it's the ones for test blocks that we are interested in for clerk)